internal/controllers/reconciliation/integration_test.go

```diff
@@ -612,7 +620,7 @@ func TestMidSynthesisDeletion(t *testing.T) {
 	comp.Finalizers = []string{"eno.azure.io/cleanup"}
 	comp.Spec.Synthesizer.Name = syn.Name
 	require.NoError(t, upstream.Create(ctx, comp))
-	comp.ResourceVersion = "" // forget this version so the next write guarantees sync'd informer
+	comp.ResourceVersion = "1" // forget this version so the next write guarantees sync'd informer
```

Comment:
empty strings aren't valid resource versions, so server returns a 400 instead of 409